fcd_weight_scale_mrp/models/res_company.py

```diff
@@ -8,3 +8,4 @@ class ResCompany(models.Model):
 
     weight_scale_username_id = fields.Many2one('res.users',string='Username')
     weight_scale_password = fields.Char(string='Password')
+    weight_scale_limit = fields.Float(string='Limit Kg', default=200)
```

Comment:
Remember this field belongs to company, so label should be more specific ("Limit kg" is very out of context). That simplified label has sense e.g. in res.config.settings form, because it will probable be placed in a section that talks a about scales configurations